src/api/worker/rest/RestClient.js

```diff
@@ -133,6 +141,18 @@ export class RestClient {
 		}
 	}
 
+	/**
+	 * Checks if the request body is too large.
+	 * Ignores the method because GET requests etc. should not exceed the limits neither.
+	 * This is done to avoid making the request, because the server will return a PayloadTooLargeError anyway.
+	 * */
+	_checkRequestSizeLimit(path: string, method: HttpMethodEnum, body: ?string | ?Uint8Array) {
+		const limit = REQUEST_SIZE_LIMIT_MAP.get(path) || REQUEST_SIZE_LIMIT_DEFAULT
+		if (body && body.length > limit) {
```

Comment:
I have some concerns about using string length as a size but I think it should work fine for most cases

Reply:
Yes, me too. What do you suggest to use instead? byteLength in case it is a Uint8Array?

Reply:
I'm not worried about Uint8Array, byteLength == length for it anyway, I'm concerned about strings but as it is only pre-check, I think it's fine. It's not like we would prevent user from sending bigger data, we would rather send bigger data than we would like in some cases.
As I merged it, I suggest leaving this as-is, just wanted to point out my concern.